Wrappers/Python/cil/optimisation/functions/L1Sparsity.py

```diff
@@ -41,20 +41,20 @@ class L1Sparsity(Function):
     
     Parameters
     ---------
-    Q: orthogonal Operator, default = Identity operator 
+    Q: orthogonal Operator 
         Note that for the correct calculation of the proximal the provided operator must be orthogonal 
-    b : Data, DataContainer, defualt is None 
+    b : Data, DataContainer, default is None 
     weight: array, optional, default=None
-        weight array matching the size of the range of operator :math:`Q`.
+        positive weight array matching the size of the range of operator :math:`Q`.
```

Comment:
nonnegative?

Reply:
Every instance of "positive weight" can be replaced with "non-negative weight". Unfortunately I couldn't find a source for this.